module1.js

```diff
@@ -28,7 +29,11 @@ function question1(name1) {
 */
 
 function question2(number1, number2) {
-  // TODO:
+  if(number1 === number2){
```

Comment:
Same here, if he function is meant to return a boolean, how can you condense it to one line?